### ISSUE TITLE
MyStats panel data question

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdNauseam#1717]
I notice that the data in the stats panel is for only the selected time period (slider values), but I don't think this is fully clear to users who may not have adjusted that slider, or even know that it exists. So I think we need to do one of the following:
1. Make this clear in the stats panel ('stats for x/y total ads'), or
2. Have the panel reflect the user's full set of ads, regardless of date, or
3. Give the user an toggle to choose between the 2 views

My feeling is that the 2nd option is preferable, as the stats will both be more representative (larger sample size) and more impressive. @cqx931 maybe you can feedback on the thinking here when you implemented this

![image](https://user-images.githubusercontent.com/737638/90721480-71a43400-e2eb-11ea-9136-1b60e692f1a0.png)

**Comment by cqx931** [comment]
The current implementation is based on the ad set mechanism in vault which is the main reason why it goes with the slider/viewable ads. I also think it's more interesting to see For 2nd option, we will need to calculate the stats in the core and pass it to the vault later.

So how about something like this:
1. Default(without moving the slider) showing stats from all ads(this should be provided by core.js)
2.When slider is adjusted, the stats will match with the current visible adset. 'stats for x/y total ads' will be added to the panel to make this clear.

**Comment by dhowe** [comment]
This sounds good. But do we really need core to show stats on the full set -- isn't the full admap available in the adview ui ?

**Comment by cqx931** [comment]
Yes you are right, see the latest commit: https://github.com/dhowe/AdNauseam/commit/6d025ab121c5726a18cc34ad97339bbbfe20041f, no need to change core.js.

For scenario 2 it looks like:
<img width="463" alt="image" src="https://user-images.githubusercontent.com/3056154/90822278-306b4e80-e302-11ea-85b5-92d516fabecb.png">
